torchaudio/datasets/vctk.py

```diff
@@ -91,7 +91,7 @@ def __init__(
             )
 
         walker = walk_files(
-            self._path, suffix=self._ext_audio, prefix=False, remove_suffix=True
+            self._path, suffix=self._ext_audio, prefix=False, remove_suffix=True, ignore=["p315"]
```

Comment:
We don't need to modify `walk_files`, since we can just use filter, as done [here](https://github.com/pytorch/audio/blob/master/torchaudio/datasets/speechcommands.py#L67).